Makefile

```diff
@@ -18,6 +18,9 @@ docker-test:
 	${CONTAINER_ENGINE} build . -f $(OPERATOR_DOCKERFILE).test -t $(OPERATOR_IMAGE_URI_TEST)
 	${CONTAINER_ENGINE} run -t $(OPERATOR_IMAGE_URI_TEST)
 
+generated-srcs:
```

Comment:
Unless there's some reason you need a target of this exact name, boilerplate [already has](https://github.com/openshift/boilerplate/blob/master/boilerplate/openshift/golang-osd-operator/standard.mk#L137) a `go-generate` target for this

Reply:
Cool, will use that.